library/src/interactor_impl.cxx

```diff
@@ -198,6 +198,14 @@ class interactor_impl::internals
         self->Options.toggle("interactor.trackball");
         render = true;
         break;
+      case 'L':
+      {
+        const double intensity = self->Options.getAsDouble("render.light.intensity");
+        const double factor = rwi->GetShiftKey() ? 0.95 : 1.05;
```

Comment:
I'm not sure the values generated by this are the best we could have, eg, pressing the key three times will set the value to .857375. 

increase/decrease of 0.05 mayb be better ?

Reply:
True, on the other hand, increasing by a fixed percentage point (eg. `+= 0.05` instead of `*= 1.05`) feel quite slow after the first few increments, and quite fast going down.
Can you try it with and see what you think?
```
const double offset = rwi->GetShiftKey() ? -0.05 : +0.05;
self->Options.set("render.light.intensity", intensity + offset);
```

Maybe there's some magic math to produce a nicer curve

Reply:
I will try your branch

Reply:
How about having the values increasing/decreasing as follow?
- `0.00, 0.01, ... 0.48, 0.49,` (by 0.01 until 0.5)
- `0.50, 0.52, ... 0.96, 0.98,` (by 0.02 until 1)
- `1.00, 1.05, ... 4.90, 4.95,` (by 0.05 until 5)
- `5.00, 5.10, ... 9.80, 9.90,` (by 0.10 until 10)
- `10.00, 10.25, 10.50, ...` (by 0.25 above 10)

Reply:
LGTM

Reply:
@Meakk

Reply:
looks good to me too